Corale.Colore/Core/Chroma.cs

```diff
@@ -289,6 +270,46 @@ public static bool IsSdkAvailable()
         }
 
         /// <summary>
+        /// Initializes the SDK if it hasn't already.
+        /// </summary>
+        /// <remarks>
+        /// Manually modifying the SDK init state is <b>untested</b>
```

Comment:
As discussed, it would be good if we can make this a tiny bit clearer to suggest that the SDK init state is untested when a developer *uninitializes* and then attempts to set an effect.